jumanji/environments/routing/connector/utils.py

```diff
@@ -310,3 +310,32 @@ def get_adjacency_mask(grid_shape: tuple, coordinate: jax.Array) -> jax.Array:
     final_mask = mask.at[neighbors[:, 0], neighbors[:, 1]].set(valid)
 
     return final_mask
+
+
+def get_correction_mask(
```

Comment:
Minor thing can we put this in the legacy generator file, otherwise it might be confusing which correction method we're using in the env